.github/workflows/scheduled_mdlinkcheck.yml

```diff
@@ -23,11 +23,14 @@ jobs:
       with:
         title: Remove dead links from repository
         content-filepath: ./.github/ISSUE_TEMPLATE/found-dead-links.md
+        labels: Documentation
```

Comment:
should we move this file somewhere else? in the issue_template directory, when someone creates an issue an option to create a dead link appears in the choices and I don't think we want this type to be an option.

Reply:
I'm about to move this file, but before I do, should we leave it here so that people can report dead links if they find them for whatever reason?